### DISCUSSION TITLE
After installing on local, how to setup migrations and how to start backend server??

### DISCUSSION THREAD

**Discussion by stockarea** [openshiporg/openship#14] [General]
I can't find in the initial documentation where it is mentioned how to start backend server, setup database etc. and as soon as I start frontend server, I am getting error of tables not existing.. Can somebody help.

**Comment by junaid33** [comment]
Hey, I have pushed an update that will handle the migrations when you run yarn dev. Keystone used to handle this in previous versions, but the new update has removed this.
